assignments/objects.js

```diff
@@ -11,54 +11,108 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: 'Example',
+  email: 'examples@you.edu',
+  gender: 'F'
+};
 
 // Write your intern objects here:
 
+function Intern(id, email, name, gender) {
+  this.id = id;
+  this.email = email;
+  this.name = name;
+  this.gender = gender;
+}
+
+const intern1 = new Intern(1, 'mmelloy0@psu.edu', 'Mitzi', 'F');
+const intern2 = new Intern(2, 'kdiben1@tinypic.com', 'Kennan', 'M');
+const intern3 = new Intern(3, 'kmummery2@wikimedia.org', 'Keven', 'M');
+const intern4 = new Intern(4, 'gmartinson3@illinois.edu', 'Gannie', 'M');
+const intern5 = new Intern(5, 'adaine5@samsung.com', 'Antonietta', 'F');
 
-// ==== Challenge 2: Reading Object Data ==== 
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(intern1.name);
 
 // Kennan's ID
+console.log(intern2.id);
 
 // Keven's email
+console.log(intern3.email);
 
 // Gannie's name
+console.log(intern4.name);
 
 // Antonietta's Gender
+console.log(intern5.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function() {
+  return 'Hello, my name is ' + this.name + '.';
```

Comment:
Could use template literals here `Hello, my name is ${this.name}.` (Make sure you wrap that in back-ticks, I can't get them to show up on here for some reason.